compare258.c

```diff
@@ -109,8 +109,10 @@ static inline uint32_t compare256_unaligned_32_static(const unsigned char *src0,
     uint32_t len = 0;
 
     do {
-        uint32_t sv = *(uint32_t *)src0;
-        uint32_t mv = *(uint32_t *)src1;
+        uint32_t sv;
+        uint32_t mv;
```

Comment:
As both variables have same type and they are not assigned a value that depend on each other, they can be declared on same line. Only when the variables are members of a struct, they should be declared on separate lines.